docs/debug.md

```diff
@@ -136,25 +136,14 @@ The syslog is particularly useful since kernel logs, launcher output, service
 status, system logs and confinement violations (these are covered in depth
 elsewhere) all get logged there.
 
-Further, systemd has log dump facility to look at all system and snap
-services. The "snappy service" command provides an easy way to monitor and
-inspect snap services. To get the status of all snap services run "snappy
-service status". To inspect the logs run "snappy service logs". Both commands
-can take a snap name to limit to a specific snap (e.g. for the service
-in `shout.sergiusens` run: `sudo snappy service logs shout`).
-
```

Comment:
Following @sergiusens request, I reported the bug here: https://bugs.launchpad.net/snapcraft/+bug/1576775
I'm going to link this PR so we can easily revert this change later.